### PR TITLE
fix: deploy registry url

### PR DESCRIPTION
**Pull request by peterambrus3**
fix: deploy registry url
feat(crux-ui): google registry handle regional server prefix

### CODE REVIEW

File: web/crux/src/app/registry/guards/registry.auth.validation.guard.ts [hunk 1]
```diff
@@ -134,9 +134,16 @@ export class RegistryAccessValidationGuard implements CanActivate {
       })
       .pipe(
         mergeMap(res => {
-          const groups = res.data as any[]
-          if (res.status !== HttpStatus.OK || groups.length < 1) {
+          const groups = res.data.map(it => it.path == req.imageNamePrefix) as any[]
+          console.log(groups)
```
Comment (robot9706) [inline web/crux/src/app/registry/guards/registry.auth.validation.guard.ts:138]: `console.log` :)

**Review by balanceee — APPROVED** [review]
LGTM

**Review by robot9706 — APPROVED** [review]
😎

**Review by polaroi8d — APPROVED** [review]
LGTM

**Pull request merged**